src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -422,6 +414,8 @@ private void Browser_Loaded(object sender, RoutedEventArgs e)
         {
             string msg = "Browser Loaded";
             LogToDynamoConsole(msg);
+
+            InitializeAsync();
```

Comment:
We initialize webview2 only after it is loaded as part of the visual tree (Browser_Loaded)